tests/integration/states/test_pip_state.py

```diff
@@ -37,13 +37,34 @@
 import salt.utils
 import salt.utils.win_dacl
 import salt.utils.win_functions
+import salt.utils.win_runas
 from salt.modules.virtualenv_mod import KNOWN_BINARY_NAMES
 from salt.exceptions import CommandExecutionError
 
 # Import 3rd-party libs
 import salt.ext.six as six
 
 
+def can_runas():
+    '''
+    Detect if we are running in a limited shell (winrm) and are un-able to use
+    the runas utility method.
+    '''
+    if salt.utils.is_windows():
+        try:
+            salt.utils.win_runas.run_as(
+                'cmd.exe /c echo 1', 'noexistuser', 'n0existp4ss',
+            )
+        except WindowsError as exc:  # pylint: disable=E060
```

Comment:
Looks like the linter doesn't like `E060`. You should be able to replace that just with `undefined-variable` instead.